drivers/at/at.c

```diff
@@ -88,18 +96,54 @@ static void _isrpipe_write_one_wrapper(void *_dev, uint8_t data)
 #endif
 }
 
-int at_dev_init(at_dev_t *dev, uart_t uart, uint32_t baudrate, char *buf, size_t bufsize)
+int at_dev_init(at_dev_t *dev, at_dev_init_t const *init)
```

Comment:
Argument list was getting too long.

Reply:
I think a long argument list is less visible. Especially when it comes to similar-typed arguments - we have to buffers now.